packages/core/src/__tests__/ceramic.test.ts

```diff
@@ -122,11 +123,42 @@ describe('Ceramic integration', () => {
     await tmpFolder.cleanup()
   })
 
+  it('can create Ceramic instance on default network', async () => {
+    const stateStorePath = await tmp.tmpName()
+    const ceramic = await Ceramic.create(ipfs1, {stateStorePath})
+    await delay(1000)
```

Comment:
My guess is that it has something to do with `ceramic.close()` below and that things are not getting shut down correctly.

Reply:
It's likely some listener such as a socket or file handler not being closed, https://jestjs.io/docs/en/cli#--detectopenhandles might help debug this

Reply:
@oed @stbrody this is not related to the `ceramic.close()` but with IPFS instance start/stop(s). I have to investigate this issue more but I'll approve this PR since it's a blocker.